test/__fixtures__/simple-monorepo-pnpm/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "monorepo-pnpm",
+  "private": true
+}
```

Comment:
this test is identical to `simple-monorepo` except that it contain `pnpm-workspace.yaml`